unlock-app/src/components/interface/locks/CheckoutUrl/index.tsx

```diff
@@ -411,9 +442,13 @@ export const CheckoutUrlPage = () => {
                   description:
                     'Customize the checkout modal interaction & additional behavior',
                   disabled: !hasSelectedConfig,
+                  loading,
                   children: (
                     <BasicConfigForm
-                      onChange={onBasicConfigChange}
+                      onChange={(fields: any, isValid = true) => {
+                        if (!isValid) return // trigger update form only when is valid to prevent update auto-update with invalid values
```

Comment:
Please add a logging statement when the config is invalid as it should not happen.

Reply:
the form already have validation with error messages in place, need to avoid to call the endpoint to save the config in that case.

<img width="500" alt="Screenshot 2023-07-25 at 10 46 03" src="https://github.com/unlock-protocol/unlock/assets/20865711/82adca4a-2859-4ff2-a017-de0352842f5f">

Reply:
I understand but please still add the logging statement so we have something to look at if for some the reason the UI does not show an error.